DOCUMENTATION.md

```diff
@@ -1,98 +1,23 @@
-# GOV.UK Notify Java client
-
-This documentation is for developers interested in using this Java client to integrate their government service with GOV.UK Notify.
-
-## Table of Contents
-
-* [Installation](#installation)
-* [Getting started](#getting-started)
-* [Send messages](#send-messages)
-* [Get the status of one message](#get-the-status-of-one-message)
-* [Get the status of all messages](#get-the-status-of-all-messages)
-* [Get a template by ID](#get-a-template-by-id)
-* [Get a template by ID and version](#get-a-template-by-id-and-version)
-* [Get all templates](#get-all-templates)
-* [Generate a preview template](#generate-a-preview-template)
-* [Get all received text messages](#get-all-received-text-messages)
-
-## Installation
-
-### Maven
-
-The notifications-java-client is deployed to [Bintray](https://bintray.com/gov-uk-notify/maven/notifications-java-client). Add this snippet to your Maven `settings.xml` file.
-
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
-
-```xml
-<?xml version='1.0' encoding='UTF-8'?>
-<settings xsi:schemaLocation='http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd' xmlns='http://maven.apache.org/SETTINGS/1.0.0' xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'>
-<profiles>
-	<profile>
-		<repositories>
-			<repository>
-				<snapshots>
-					<enabled>false</enabled>
-				</snapshots>
-				<id>bintray-gov-uk-notify-maven</id>
-				<name>bintray</name>
-				<url>http://dl.bintray.com/gov-uk-notify/maven</url>
-			</repository>
-		</repositories>
-		<pluginRepositories>
-			<pluginRepository>
-				<snapshots>
-					<enabled>false</enabled>
-				</snapshots>
-				<id>bintray-gov-uk-notify-maven</id>
-				<name>bintray-plugins</name>
-				<url>http://dl.bintray.com/gov-uk-notify/maven</url>
-			</pluginRepository>
-		</pluginRepositories>
-		<id>bintray</id>
-	</profile>
-</profiles>
-<activeProfiles>
-	<activeProfile>bintray</activeProfile>
-</activeProfiles>
-</settings>
-```
-Then add the Maven dependency to your project.
-```xml
-    <dependency>
-        <groupId>uk.gov.service.notify</groupId>
-        <artifactId>notifications-java-client</artifactId>
-        <version>3.9.1-RELEASE</version>
-    </dependency>
-
-```
-</details>
-
-### Gradle
-
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
-
-```
-repositories {
-    jcenter()
-}
-
-dependencies {
-    compile('uk.gov.service.notify:notifications-java-client:3.9.1-RELEASE')
-}
-```
-</details>
-
-### Artifactory or Nexus
-
-Click 'set me up!' on https://bintray.com/gov-uk-notify/maven/notifications-java-client for instructions.
-
-## Getting started
+# Java client documentation
+
+This documentation is for developers interested in using GOV.UK Notify to send emails, text messages or letters.
+
+# Set up the client
+
+## Install the client
+
+The `notifications-java-client` deploys to Bintray.
```

Comment:
The `notification-java-client` is deployed to Bintray.

Reply:
Changed